calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/views/initialization_view.py

```diff
@@ -75,6 +85,49 @@ def __init__(self, calibrator: "CameraIntrinsicsCalibratorUI", cfg):  # noqa F82
         source_layout.addWidget(self.data_source_combobox)
         self.source_group.setLayout(source_layout)
 
+        self.params_combobox = QComboBox()
+        self.params_combobox.addItem("General", 0)
+        self.params_combobox.addItem("C1", 1)
+        self.params_combobox.addItem("C2", 2)
+        self.params_combobox.addItem("Ceres Calib", 3)
+        self.params_combobox.addItem("Load File", 4)
+
+        def on_params_combo_box_changed(index):
+            if self.params_combobox.currentText() == "Load File":
+                file_name, _ = QFileDialog.getOpenFileName(
+                    self, "Open File", "", "All Files (*.*);;Text Files (*.yaml)"
+                )
+                if file_name:
+                    print(f"Selected file: {file_name}")
+                    config_file_path = file_name
+            elif self.params_combobox.currentText() == "C1":
```

Comment:
This if else is not a good design. There are several examples in this package of how to address this cases

Reply:
ok, i will try to change the design

Reply:
Change the design, so will load the yaml files from te config directory

Reply:
Whenever you address a comment, please reference the commit here. These commits should be as small as possible, so the review process go smoother

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e